earth_enterprise/src/server/wsgi/wms/ogc/common/configs.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#
+# Copyright 2017 Google Inc.
```

Comment:
Do these copyright notices need to be updated to 2019 and The OpenGEE Contributors?

Reply:
Wasn't sure when I was making the changes, most of the Copyrights have the same year and name. Do these ones need to change because it's a new file?

Reply:
Since this appears to be a new file, then yes

Reply:
If it was just copied from a different directory, then no

Reply:
This file was copied from fusion/portableglobe/cutter.